lib/asciidoctor/latex/node_processors.rb

```diff
@@ -653,6 +641,10 @@ def image_process
         width = '2.5truein'
       end
       raw_image = self.attributes['target']
+      imagesdir = "#{document.attributes['imagesdir']}"
+      unless imagesdir.empty?
```

Comment:
Alternately, you can write this as:

```ruby
unless (imagesdir = document.attr 'imagesdir').nil_or_empty?
  raw_image = ::File.join imagesdir, raw_image
end
```

Every object has a `nil_or_empty?` method in an Asciidoctor environment.